requirements.txt

```diff
@@ -1,10 +1,7 @@
-blinker==1.9.0
-click==8.1.8
-Flask==3.1.0
-flask-cors==5.0.1
-itsdangerous==2.2.0
-Jinja2==3.1.5
-MarkupSafe==3.0.2
-python-dotenv==1.0.1
-Werkzeug==3.1.3
-hackerbot
\ No newline at end of file
+hackerbot
+fastapi
```

Comment:
I'd recommend setting version numbers here. At a later date, I would suggest considering something like [uv](https://github.com/astral-sh/uv) or generally following the new spec for [pylock.toml](https://packaging.python.org/en/latest/specifications/pylock-toml/#pylock-toml-spec).

Note: I haven't personally made the switch yet on my projects but I typically use pipenv which has some of the benefits of switching.